### ISSUE TITLE
Ability to open res files

### ISSUE THREAD

**Issue by PeterKottas** [gilmarsquinelato/i18n-manager#397]
It would be awesome if you could open res files frequently used to translate dot net (core) apps.

**Comment by gilmarsquinelato** [comment]
Hello @PeterKottas,

This request was already made, but due to the structure of .res files not being as simple as the .json and .yaml files, it gets harder to consolidate the translations and after to save them back to the files.

**Comment by HiiZun** [comment]
Hello, this feature can be very helpful for my situation, i'm using js files to save translations,
files can be formed like this:
`KEY: "My value"`
or with variables:
![image](https://user-images.githubusercontent.com/44111470/84541854-81563980-acf8-11ea-8623-427deb4f8c94.png)

**Comment by gilmarsquinelato** [comment]
@HiiZun how this feature would be related to js files?

**Comment by PeterKottas** [comment]
@gilmarsquinelato thanks for getting back to me. I understand. It will probably require some import/export modules as well as an internal data structure that would keep the imported model in memory. Maybe something cool for the future ;)

**Comment by gilmarsquinelato** [comment]
Yeah, I'll need to do some researches about this way to treat the files with more "complex" data.
Maybe in the future we can deliver that one :)

**Comment by PeterKottas** [comment]
Cool, I think there would be a lot of value in this!
